fastlane/Jetpack-Fastfile

```diff
@@ -156,41 +156,33 @@ lane :update_jetpack_appstore_strings do |options|
 end
 
 #####################################################################################
-# download_jetpack_localized_metadata
+# download_jetpack_localized_app_store_metadata
 # -----------------------------------------------------------------------------------
-# Downloads localized metadata for App Store Connect from GlotPress.
+# Downloads localized metadata (intended for App Store Connect) from GlotPress for Jetpack.
 # -----------------------------------------------------------------------------------
 # Usage:
-# bundle exec fastlane download_localized_metadata
+# bundle exec fastlane download_jetpack_localized_app_store_metadata
 #####################################################################################
-desc 'Downloads localized metadata for App Store Connect from GlotPress'
+desc 'Downloads localized Jetpack metadata (for App Store Connect) from GlotPress'
 lane :download_jetpack_localized_app_store_metadata do
```

Comment:
While working on this PR, I really felt the pain of those being in different files while they would be related to the same type of action.

While I like the idea of splitting our otherwise-super-large `Fastfile`, I feel that the way we did the split (WP vs JP) was not the best of all, compared to theme-oriented split depending on what category of work the actions are for (release management top-level lanes, vs localization, vs ASC, etc).

See PR description for some ideas about that, though that's definitively for a separate PR. But a separate PR that I'm more and more tempted to address sooner rather than later, especially if I'm gonna continue having to work on the `Fastfiles` while continuing the work on that L10n tooling migration and other related improvements.